specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Is it "kosher" for us to re-tabulate the entire election ourselves? (it's also unclear where exactly we'd do that; we don't have a database table with individual columns for the individual fields of a CVR - eliminating the easy "ask the database for the sum of these columns" method of counting - so it'd have to be done by the Java code in memory somewhere and stored... where?)